src/logic/logic.ts

```diff
@@ -149,8 +160,48 @@ function readPushByteOp(
   return [size, byteArray];
 }
 
+/** sanityCheckProgram performs heuristic program validation:
+ * check if passed in bytes are Algorand address or is B64 encoded, rather than Teal bytes
+ *
+ * @param program - Program bytes to check
+ * @throws
+ * @returns
```

Comment:
nit: `@throws` and `@returns` should be removed if you don't wish to add explanation to them.

Reply:
good point, removing